### ISSUE TITLE
Feature Request: Ticket Barcode Generation orientation.

### ISSUE THREAD

**Issue by thebeline** [eventespresso/event-espresso-core#185]
It would be swell to be able to generate the barcode 90 degrees rotated.

My thought was to have this trigger if the line width is greater than the line height, ie:

* It wouldn't make any sense to have a barcode who's lines were `70px` wide and `1px` tall, so the generator flips the image and generates vertically.

Whether it is `+90deg` or `-90deg` does not matter for a barcode scanner.

**Comment by thebeline** [comment]
**This is me being punchy, not actually an a-hole: **

You should get your repos straight, it's actually an issue for the Ticketing Module
* `eea-ticketing/EE_Ticketing.class.php`

;-)

**Comment by nerrad** [comment]
you are absolutely right.  It's less an issue with me getting my repos straight and more dealing with a bunch of issues at once and not multi-tasking very well.  So its time for me to buckle down and focus on one thing at a time.  I will address this at some point but it may not be soon.

**Comment by thebeline** [comment]
NP mate, do what you have to do, I imagine you have a lot on your plate.  Thanks for your work!

**Comment by nerrad** [comment]
Appreciate the understanding.  Appreciate the suggestions, feedback and pull requests you are making, however with it being the holiday season its unlikely there will be quick action on any of them.  Rest assured it will be reviewed and feedback will be given :)  we *do* appreciate any and all contributions.
